fulltext/worker.py

```diff
@@ -1,6 +1,53 @@
 """Initialize the Celery application."""
+from typing import Any
+from celery.signals import task_prerun, celeryd_init, worker_init
 
+import docker
+from docker.errors import ImageNotFound, APIError
+
+from arxiv.vault.manager import ConfigManager
 from fulltext.factory import create_worker_app, celery_app
 
 app = create_worker_app()
```

Comment:
It gives us an application context (L12, ``app.app_context().push()``) so that LocalProxies work correctly. E.g. service instances that are bound to ``g``.